addons/field_rations/functions/fnc_consumeItem.sqf

```diff
@@ -70,11 +72,15 @@ private _soundPlayed = if (_consumeAnim != "" && {vehicle _player == _player &&
 
 private _fnc_onSuccess = {
     params ["_args"];
-    _args params ["_player", "_consumeItem", "_replacementItem", "_thirstQuenched", "_hungerSatiated"];
+    _args params ["_player", "_consumeItem", "_replacementItem", "_thirstQuenched", "_hungerSatiated", "", "", "", "_isMagazine"];
     TRACE_1("Consume item successful",_args);
 
     // Remove consumed item
-    _player removeItem _consumeItem;
+    if (_isMagazine) then {
+        _player removeMagazineGlobal _consumeItem;
```

Comment:
I don't think we need global version because player should always be local
but I guess it doesn't hurt

Reply:
I wasn't sure what "local deletion" means so I wanted to be safe.